apps/challenges/models.py

```diff
@@ -226,6 +231,13 @@ def create_eks_cluster_for_challenge(sender, instance, created, **kwargs):
     aws.challenge_approval_callback(sender, instance, field_name, **kwargs)
 
 
+@receiver(signals.post_save, sender="challenges.Challenge")
+def github_sync_challenge(sender, instance, created, **kwargs):
+    if instance.github_repository and instance.github_token:
+        serialized_obj = serializers.serialize("json", [instance])
+        github_challenge_sync.delay(serialized_obj)
```

Comment:
@savish28 the method names are confusing `github_challenge_sync` and `github_sync_challenge` (post save hook) it is not clear what these methods are for. Please rename the methods.

Reply:
Updated.